release-content/0.15/release-notes/14663_shader_storage_buffer.md

```diff
@@ -0,0 +1,38 @@
+A new asset `ShaderStorageBuffer` has been added to simplify working with storage buffers in custom materials and compute shaders as part of our ongoing work to make working with compute simpler.

```

Comment:
Can you give me one sentence explaining what a storage buffer is here?

Reply:
Gave you more than one sentence, but realized education is important here because people might not even know why this would be useful.